docs/source/notebook-components/weaver_example.ipynb

```diff
@@ -10,25 +10,40 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        if resp.status_code == 404:
</pre><p>We want this to explicitly fail if expected birds are missing. This would mean the service failed to initialize properly. If additional ones should be ignored due to custom definition, <code>WEAVER_TEST_KNOWN_BIRDS</code> must be used.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/pavics-sdi/pull/359/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>